packages/backend/src/graphql/mutations/update-flow-transfer-status.ts

```diff
@@ -111,6 +111,19 @@ const updateFlowTransferStatus = async (
       )
     }
 
+    // additional update: set tiles actions to incomplete too
+    const numIncompleteForTiles = await Step.query(trx)
+      .patch({ status: 'incomplete' })
+      .where('flow_id', flow.id)
+      .andWhere('app_key', 'tiles')
+
+    // sanity check
+    if (numIncompleteForTiles > flow.steps.length) {
+      throw new Error(
+        'Update tiles status query went wrong, please contact plumber@open.gov.sg for more support',
+      )
+    }
+
```

Comment:
M365 too, pls. Can refactor to a callback or a IApp attribute (e.g. `shouldSetStepToIncompleteAfterTransfer`) later.

```suggestion
    // additional update: set tiles actions to incomplete too
    const numIncompleteForTiles = await Step.query(trx)
      .patch({ status: 'incomplete' })
      .where('flow_id', flow.id)
      .andWhere('app_key', 'tiles')

    // sanity check
    if (numIncompleteForTiles > flow.steps.length) {
      throw new Error(
        'Update tiles status query went wrong, please contact plumber@open.gov.sg for more support',
      )
    }
    
    // additional update: set M365 actions to incomplete too
    const numIncompleteForExcel = await Step.query(trx)
      .patch({ status: 'incomplete' })
      .where('flow_id', flow.id)
      .andWhere('app_key', 'm365-excel')

    // sanity check
    if (numIncompleteForExcel > flow.steps.length) {
      throw new Error(
        'Update M365 status query went wrong, please contact plumber@open.gov.sg for more support',
      )
    }

```

Reply:
oh but m365 has a connection and I'm already setting the status to be incomplete for all steps with connections?

Reply:
oh ya. my bad

Reply:
ok thksss